spec.bs

```diff
@@ -1328,17 +1360,22 @@ To <dfn>generate and score bids</dfn> given an [=auction config=] |auctionConfig
               1. If [=query component ad k-anonymity count=] given |adComponent|'s
                 [=interest group ad/render url=] returns true, [=list/append=] |adComponent| to |ig|'s
                 [=interest group/ad components=].
+          1. If |perBuyerCumulativeTimeout| is less than |perBuyerTimeout|:
+            1. Set |perBuyerTimeout| to |perBuyerCumulativeTimeout|.

```

Comment:
May be good to explicitly handle negative (non-positive?) timeouts in =evaluate a script=?

Reply:
after the change above, seems no need to do so since =evaluate a script= will no longer get negative timeouts? 0 seems fine in current [=evaluate a script=](https://github.com/WICG/turtledove/blob/main/spec.bs#L2395).

Reply:
Maybe? It's not like "immediately" is a real thing in computing, so even if it's like 1 nanosecond, you can end up with generateBidDuration actually being 1ms... or, well, it depends on how exactly rounding for the intervals work. Like if you have 4 0.5ms intervals, is it gonna add up to 2ms, or 0ms?

Reply:
done. Added a check in [=evaluate a script=].